### PR TITLE
Fixing Grammar Mistakes in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Mitulsharma123**
- Fixing typo, punctuation marks 
- Added correct word

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ Please follow this style to make openshift-preflight easy to review, maintain an
 
 ### Commit Message Format
 
-We follow a rough convention for commit messages that is designed to answer two
+We follow a rough convention for committing messages that are designed to answer two
```
Comment (bcrochet) [inline CONTRIBUTING.md:53]: "commit messages" is the correct verbage. "is" is also correct here. If you take out "for commit messages", the sentence reads "We follow a rough convention that is designed..."

**Review by bcrochet — CHANGES_REQUESTED** [review]
Take out one of the changes.

**Comment by bcrochet** [comment]
Please do not include merge commits. Squash down to a single commit please.

File: CONTRIBUTING.md (hunk 1 above)
Comment (Mitulsharma123) [reply CONTRIBUTING.md]: @bcrochet Corrected to original statement

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -19,10 +19,10 @@ If any part of the openshift-preflight project has bugs or documentation mistake
 
 To make the bug report accurate and easy to understand, please try to create bug reports that are:
 
-- Specific. Include as much details as possible: which version, what environment, what configuration, etc.
+- Specific. Include as many details as possible: which version, what environment, what configuration etc.
```
Comment (bcrochet) [inline CONTRIBUTING.md:22]: The final comma should stay.

**Review by bcrochet — CHANGES_REQUESTED** [review]
Please also squash to a single commit.

File: CONTRIBUTING.md (hunk 2 above)
Comment (Mitulsharma123) [reply CONTRIBUTING.md]: has been added back.

**Review by bcrochet — APPROVED** [review]
/lgtm

**Review by acornett21 — APPROVED** [review]

**Pull request merged**